ffi/src/lib.rs

```diff
@@ -42,6 +42,8 @@ pub fn mitmproxy_rs(_py: Python, m: &PyModule) -> PyResult<()> {
     m.add_class::<server::WireGuardServer>()?;
     m.add_function(wrap_pyfunction!(util::genkey, m)?)?;
     m.add_function(wrap_pyfunction!(util::pubkey, m)?)?;
+    #[cfg(target_os = "macos")]
+    m.add_function(wrap_pyfunction!(util::load_cert, m)?)?;
```

Comment:
As discussed: One downside of adding `cfg(target_os=...)` here is that the function won't render in the docs. Let's do the same we do with `start_os_proxy` and add the `#[cfg(...)]` block within the function. 😃